engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/callcaching/CallCacheDiffQueryParameter.scala

```diff
@@ -29,11 +31,19 @@ object CallCacheDiffQueryParameter {
       }
     }
     
-    val workflowAValidation = extractAttribute("workflowA")
-    val workflowBValidation = extractAttribute("workflowB")
+    def validateWorkflowId(parameter: String) = for {
+      workflowIdString <- extractAttribute(parameter)
+      workflowId <- Try(WorkflowId.fromString(workflowIdString.trim)) match {
+        case Success(id) => id.validNel
+        case Failure(f) => f.getMessage.invalidNel
```

Comment:
@danbills Even with `fromTry(Try(WorkflowId.fromString(workflowIdString.trim))).toValidatedNel` I get a `Validated[NonEmptyList[Throwable]], WorkflowId]` and I want a `Validated[NonEmptyList[String]], WorkflowId]`, do you know how I can do that ?